Lib9c/Model/State/GameConfigState.cs

```diff
@@ -65,12 +65,16 @@ public override IValue Serialize()
                 [(Text) "daily_arena_interval"] = DailyArenaInterval.Serialize(),
                 [(Text) "weekly_arena_interval"] = WeeklyArenaInterval.Serialize(),
             };
+#pragma warning disable LAA1002
             return new Dictionary(values.Union((Dictionary) base.Serialize()));
+#pragma warning restore LAA1002
         }
 
         public void Set(GameConfigSheet sheet)
         {
+#pragma warning disable LAA1002
             foreach (var row in sheet)
```

Comment:
Is it okay that we don't care iteration order of `sheet`?

Reply:
I don't think it's a problem. but, I'll change it to use `OrderedList`.